cmake/msys/msys_helper.sh

```diff
@@ -222,9 +212,8 @@ if [ ! -e "$mingw_root/lib/libgig/libgig.dll.a" ]; then
 	make install
 
 	mv "$mingw_root/lib/bin/libakai-0.dll" "$mingw_root/bin"
-	mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"
 
-	if [ $? -ne 0 ]; then
+	if ! mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"; then
```

Comment:
There's something I don't understand. Why don't we (I) add `set -e` on the previous PR on this script ? That would allow us to remove almost all these tests...